*** Makefile.linux.orig	2011-08-25 16:17:55.000000000 +0200
--- Makefile.linux	2012-12-19 22:52:13.764594991 +0100
***************
*** 5,11 ****
  M_ARCH ?= $(shell uname -m)
  ifeq (x86_64,${M_ARCH})
  LDFLAGS.EXTRA = -L/usr/X11R6/lib64
! LIBDIR = $(GLEW_DEST)/lib64
  else
  LDFLAGS.EXTRA = -L/usr/X11R6/lib
  LIBDIR = $(GLEW_DEST)/lib
--- 5,11 ----
  M_ARCH ?= $(shell uname -m)
  ifeq (x86_64,${M_ARCH})
  LDFLAGS.EXTRA = -L/usr/X11R6/lib64
! LIBDIR = $(GLEW_DEST)/lib
  else
  LDFLAGS.EXTRA = -L/usr/X11R6/lib
  LIBDIR = $(GLEW_DEST)/lib
